docs/modules/ROOT/pages/api/erc1155.adoc

```diff
@@ -305,8 +308,8 @@ Queries if `operator` is an authorized operator for `owner`.
 ==== `[.contract-item-name]#++uri++#++(self: @ContractState, token_id: u256) -> ByteArray++` [.item-kind]#external#
 
 This implementation returns the same URI for *all* token types. It relies
-on the token type ID substitution mechanism defined in the EIP:
-https://eips.ethereum.org/EIPS/eip-1155#metadata.
+on the token type ID substitution mechanism
+https://eips.ethereum.org/EIPS/eip-1155#metadata[specified in the EIP].
```

Comment:
we should probably reuse the link defined at the top. i think we can reuse the URL and display different link texts every time it's used